extensions/wikia/AdEngine/js/AdEngine2.run.js

```diff
@@ -132,18 +118,29 @@ window.AdEngine_loadLateAds = function () {
 
 	function loadLateFn() {
 		require([
-			'ext.wikia.adEngine.adConfigLate', 'ext.wikia.adEngine.adEngine', 'ext.wikia.adEngine.lateAdsQueue', 'ext.wikia.adEngine.adTracker', 'wikia.log'
-		], function (adConfigLate, adEngine, lateAdsQueue, adTracker, log) {
-			var module = 'AdEngine_loadLateAds';
+			'ext.wikia.adEngine.adConfigLate',
+			'ext.wikia.adEngine.adEngine',
+			'ext.wikia.adEngine.lateAdsQueue',
+			'ext.wikia.adEngine.adTracker',
+			'wikia.krux',
+			'wikia.log'
+		], function (adConfigLate, adEngine, lateAdsQueue, adTracker, krux, log) {
+			var module = 'AdEngine_loadLateAds',
+				kruxSiteId = 'JU3_GW1b';
+
 			log('launching late ads now', 1, module);
 			log('work on lateAdsQueue according to AdConfig2Late', 1, module);
 			adTracker.measureTime('adengine.init', 'queue.late').track();
 			adEngine.run(adConfigLate, lateAdsQueue, 'queue.late');
+
+			log('Loading Krux module, site id: ' + kruxSiteId, 'debug', 'wikia.krux');
+			krux.load(kruxSiteId);
 		});
 	}
 
 	require(['ext.wikia.adEngine.adContext', require.optional('wikia.abTest')], function (adContext, abTest) {
-		var adsAfterPageLoad = adContext.getContext().lateAdsAfterPageLoad && abTest && abTest.inGroup('ADS_AFTER_PAGE_LOAD', 'YES');
+		var adsAfterPageLoad = adContext.getContext().lateAdsAfterPageLoad &&
+			abTest && abTest.inGroup('ADS_AFTER_PAGE_LOAD', 'YES');
```

Comment:
A little clean up is possible: ADS_AFTER_PAGE_LOAD is no longer a running experiment.